### ISSUE TITLE
Tool tips are not invoked when user sets focus on "Refresh the application visual tree in inspector/ Select a UI element to inspect the running application/ Rotate left / Reset View / Rotate Right / Show frames / Show contents" for 3D View of Inspector window. 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#379]
### Steps to Reproduce
1. Launch Xamarin Workbooks application. This will invokes "Create a new Workbook" window.
2. Create a Workbook with WPF flatform.
3. Run the sample workbook code. Navigate to View Inspector tab and invoke it.
4. Navigate to "Refresh the application visual tree in inspector/ Select a UI element to inspect the running application/ Rotate left / Reset View / Rotate Right / Show frames / Show contents" of 3D View and invoke the tool-tips.
### Expected Behavior
Tool Tips should be invoked upon setting the focus on "Refresh the application visual tree in inspector/ Select a UI element to inspect the running application/ Rotate left / Reset View / Rotate Right / Show frames / Show contents" for each element.

### Actual Behavior
 Tool-tips are not invoked even after setting keyboard focus. 
These elements "Refresh the application visual tree in inspector/ Select a UI element to inspect the running application/ Rotate left / Reset View / Rotate Right / Show frames / Show contents" in 3d view of inspector not getting tool-tips  even after setting the focus.

### User Impact:
User who rely on keyboard will not be conveyed with tool-tip information.
### Screenshots:
![issue 15](https://user-images.githubusercontent.com/22956130/40761730-806d7d0e-64ba-11e8-9a0f-902e06c521ba.png)

### Environment
Windows : Version 1803 (OS Build : 17134.81)
Xamarin Version : 1.4(master/b3f92f9)

**Comment by WiproPSAccTest** [comment]
Please add the following labels to the bug:
A11yMAS
Accessibility
Xamarin-Workbooks-Windows
Wipro
MAS3.3.2

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
